policyengine_us/variables/gov/states/mo/tax/income/deductions/mo_business_income_deduction.py

```diff
@@ -0,0 +1,25 @@
+from policyengine_us.model_api import *
+
+
+class mo_business_income_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Missouri business income deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://dor.mo.gov/forms/MO-1040%20Instructions_2023.pdf#page=16",
+        "https://revisor.mo.gov/main/OneSection.aspx?section=143.022",
+    )
+    defined_for = StateCode.MO
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(
+            period
+        ).gov.states.mo.tax.income.deductions.business_income
+        qualified_business_income = person("qualified_business_income", period)
```

Comment:
can also delete line 17
```suggestion
        qualified_business_income = add(tax_unit, period, ["qualified_business_income"])
```